presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -16,7 +16,7 @@ function canonical_path() {
 
 source ${BASH_SOURCE%/*}/../../../bin/locations.sh
 
-export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-'teradatalabs/cdh5-hive:4'}
```

Comment:
@ArturGajowy : What's the advantage of defining the variable
export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-"teradatalabs/cdh5-hive:$DOCKER_IMAGE_VERSION"}
We don't have that for kerberos images